airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/RouterService.java

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
This diff shows up as a brand new file because I moved the RouterService from `/sync/ to `/scheduling/` because it's used by the ConnectionManagerWorkflow, not the SyncWorkflow.

This diff also contains the new functionality of reading the connection's Geography from the database instead of from the environment variable